src/main/scala/com/wanari/git_in_idea_basics/Main.scala

```diff
@@ -0,0 +1,5 @@
+package com.wanari.git_in_idea_basics
+
+object Main extends App {
+  println("Hello world!")
```

Comment:
Kinda bored of this message. Please change it to something more unique!
- [] change message